src/javascript/common/champion.js

```diff
@@ -133,7 +132,7 @@ const Champion = (function() {
         login: module => (
             module === MetaTrader ?
             Utility.template(`To register an MT5 account, please <a href="[_1]" class="login">log in</a> to your ChampionFX account<br />
-                Don't have a ChampionFX account? <a href="[_2]">Create one</a> now`, [`${'java'}${'script:;'}`, url_for('/')]) :
+                Don't have a ChampionFX account? <a href="[_2]" class="toggle-signup-modal">Create one</a> now`, [`${'java'}${'script:;'}`, `${'java'}${'script:;'}`]) :
```

Comment:
May use the same placeholder `[_1]` and one replacement value, as both are the same.